README.md

```diff
@@ -164,9 +163,115 @@ modelSelector | JSON | Use the Cloudant Query selector syntax to associate model
 
 ### _rev Property
 
-In a document, property `_rev` is the latest doc revision and must be provided when modifying the doc. Currently when updating/deleting a doc, the connector first retrieves that doc, then gets its `_rev`, then calls a modify api with `_rev` included. It makes three calls to the database and this would easily result in race condition.
-  
-Considering the nature of Cloudant, we plan to give user the flexibility to define `_rev` property in model and provide `_rev` by themselves instead of the connector. In such way user can get the revision and control conflict before using that revision to modify the document.
+In a document, property `_rev` is the latest doc revision and must be provided when modifying the doc.
+
+Our connector allows the user to retrieve back the `_rev` property upon all CRUD operations, however does not add it to the model definition.
+
+If you would like to have a `_rev` property on your model, as an end user, the onus is on you to add the property in the model definition.
+
+#### Example CRUD operations with `_rev`
+
+`model.json`
+``` json
+{
+  ... 
+  "properties": {
+    "_rev": {
+      "type": "string"
+    },
+    "name": {
+      "type": "string"
+    }
+  },
+  ...
+}
+```
+
+- Create
+
+```javascript
+  Model.create([{
+    name: 'Foo',
+  }, {
+    name: 'Bar',
+  }], function(err, result) {
+    if (err) throw err;
+    console.log('Created instance: ' + JSON.stringify(result));
+  });
+```
+
+**Note:** Cloudant does not allow customized `_rev` value, hence creating an instance with a `_rev` value will give an error. The onus is on the user if they fail to comply to this rule.
```

Comment:
You can warn them, but how does it result in an error if we strip the `_rev` value even if they provide it? i.e. 
```
 // strip the _rev property out since users should not update the _rev value either ways
+  delete data._rev;
```